### PR TITLE
jdk11 with 8/9 compatibility

### PR DESCRIPTION
**Pull request by michael-mclawhorn**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

PR  Testing a JDK bump

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -719,7 +721,7 @@
                     <configuration>
                         <rules>
                             <requireJavaVersion>
-                                <version>[1.8,1.9]</version>
+                                <version>[1.8,12]</version>
```
Comment (will-lauer) [inline pom.xml:724]: why is this 12, not 11?

**Review by will-lauer — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (michael-mclawhorn) [reply pom.xml:724]: because I wanted all versions of 11 to be in range, I think that's a boundary marker.

Comment (will-lauer) [reply pom.xml:724]: would `[8,12)` work to make it an open interavl?

Comment (will-lauer) [reply pom.xml:724]: [8,12] usually implies that it includes 12.

**Pull request merged**